context/contextIsShowingPanelMenu.tsx

```diff
@@ -0,0 +1,48 @@
+import {
```

Comment:
change name of file, better will be showingPanelMenu, remove the context from names like cart and others

Reply:
you put the file in folder context not need have in name